yoti-sdk-api/src/main/java/com/yoti/api/client/shareurl/DynamicScenario.java

```diff
@@ -0,0 +1,31 @@
+package com.yoti.api.client.shareurl;
+
+import java.util.List;
+import com.yoti.api.client.shareurl.extension.Extension;
+import com.yoti.api.client.shareurl.policy.DynamicPolicy;
+
+/**
+ * Data required when initiating a dynamic share
+ *
+ */
+public interface DynamicScenario {
+
+    /**
+     * The device's callback endpoint. Must be a URL relative to the Application Domain specified in your Dashboard
+     *
+     */
+    String callbackEndpoint();
+
+    /**
+     * The customisable {@link DynamicPolicy} to use in the share
+     *
+     */
+    DynamicPolicy policy();
+
+    /**
+     * List of {@link Extension} to be activated for the application
```

Comment:
Would you add an s after the Extension so it'd be `{@link Extension}s`, or would that not look right when it's generated?

Reply:
I don't think it looks right